File: 프로그래머스/2/12945. 피보나치 수/피보나치 수.py
def solution(n):
    mod = 1234567
    
    fibo = [0] * 100001
    fibo[1] = 1
    
    for i in range(2, 100001):
        fibo[i] = (fibo[i-1] + fibo[i-2]) % mod
    
    return fibo[n]

File: 프로그래머스/2/12945. 피보나치 수/피보나치 수.swift
func solution(_ n:Int) -> Int {
    
    let mod = 1234567
    
    var fibo = [Int](repeating: 0, count: 100001)
    fibo[1] = 1
    
    for i in 2..<fibo.count {
        fibo[i] = (fibo[i-1] + fibo[i-2]) % mod
    }
    
    return fibo[n]
}